pxr/imaging/hgiMetal/graphicsCmds.mm

```diff
@@ -818,6 +820,8 @@
     
     // Apple Silicon only support memory barriers between vertex stages after
     // macOS 12.3.
+    //For iOS we may want to introduve an alternative path
```

Comment:
typo, "introduce".
Is there anything that can be said about what the alternative path is? That would help people reading the code in the future understand what was intended by this comment.

Reply:
Adjusted it now - 16.0 introduced the ability to perform the operation on iOS!